docs/source/examples/styling.ipynb

```diff
@@ -0,0 +1,218 @@
vuY8qhKbSYKyFajoqhVhaTHwM3Mze84feG0FEIIc+ZmZ4ZkgqT8N/L/6XZqQnRYlQMtbI3/d7Ez6E/Q1esK3QUQshzZq5vjlndZuFSxiWcTD4pdBxCSAOoGGolqUWpuJpZNd+RnlhP4DSEkNYywWsCPDt7YtXVVShXlgsdhxCiARVDreSzq5/h3RPvorCyUOgohJBWpCPSwaIei5BWnIbdsbuFjkMI0YBGk7WC61nXcSL5BGZ1nQVTXVOh4xBCWlmwbTDWDliLvvZ9hY5CCNFAq1qGGGPDGGMxjLE4xtiiRo4bzxjjjDGtn06ec47V11bD2sAaU+VThY5DCBHIIKdBkIglUKqVQkchhNShNcUQY0wMYCOAlwHIALzKGJNpOM4EwD8AXGrdhM/mj6Q/cDvnNmZ1m0Ur0hPSwd3JuYPhvw1HbH6s0FEIIbVoTTEEIBhAHOc8gXNeCWAngNEajvsAwEoAbaInYpmyDIHSQIxyHyV0FEKIwJxMnVBcWYz119cLHYUQUos2FUP2AFJqPU59tK0GY6wbAEfO+YHGLsQYm8EYu8oYu5qTk9PySZsgzCMM3730HcQisaA5CCHCM9Mzw5v+b+J06mlat4wQLaJNxZCm+eprZiljjIkAfA5g3pMuxDnfwjkP4pwHWVlZtWDEp1euLMeRxCNQqVU0FT8hpMZkn8mw0LfA59c+p4kYCdES2lQMpQJwrPXYAUB6rccmAPwAnGaMJQJ4AcA+be1EvTt2NxacWYA7uXeEjkII0SKGEkO83eVtXM++jqtZV4WOQwiBdg2tvwLAkzHmCiANwCQAk6t3cs4LAFhWP2aMnQYwn3Oudb9NKlQV+DbiW/Sw6YGu1l2FjkMI0TLjPMfB0cQRQVKt/CxHSIejNS1DnHMlgFkAjgKIAvAz5/wuY2wFY6xN9T7eHbsbOWU5mNllptBRCCFaSCKW4EX7F+kWOiFaQptahsA5PwTgUJ1tSxs4tn9rZGqqClUFvr3zLQKlgehh00PoOIQQLfZtxLeIyI3Amv5rhI5CSIemNS1D7UVaURoMJYbUKkQIeSrHko4hIjdC6BiEdGhUDLUwt05u2Bu2F8E2wUJHIYRouVe8XoGJrgm23tkqdBRCOjQqhlpQalEqShWlEDER9QUghDyRsa4xJvtMxonkE7hfcF/oOIR0WFQMtaD3z7+PqYen0twhhJCnNslnEiQiCX6K/knoKIR0WFQMtZCovChczryMkW4jqVWIEPLULA0ssbjnYoz20LT6ECGkNWjVaLK2bEfkDhjqGGKs11ihoxBC2pgJXhOEjkBIh0YtQy3gQfkDHE48jDCPMJjqmgodhxDSBsU/jMeaa2ug5mqhoxDS4VAx1ALOpJyBUq3EK96vCB2FENJGRT2IwncR3+FC+gWhoxDS4VAx1ALGeI7B3tF74d7JXegohJA2aqjzUJjrm+PnmJ+FjkJIh0PFUAtx6+QmdARCSBumK9ZFqFsozqaeRV5ZntBxCOlQqBhqpuUXluPTy58KHYMQ0g6EeYRByZU4dP/Qkw8mhLQYKoaaoaCiAPvi9kGhVggdhRDSDnh09kCIXQj9TiGkldHQ+mb4I+kPVKorMdaThtMTQlrG5iGbhY5ASIdDLUPNcPj+YbiYusDX3FfoKISQdoRzjrTiNKFjENJhUDH0jLJKsnA18yqGuw6nGacJIS3qv5f/i8kHJ0OlVgkdhZAOgYqhZ8QYw1TZVAx3Gy50FEJIOxMoDcSD8ge4nn1d6CiEdAhUDD0ja0NrzO8xH86mzkJHIYS0M33s+0BfrI+jiUeFjkJIh0DF0DPIKsnCxYyL1IRNCHkuDCWG6OPQByeST9DvGUJaARVDz+Dw/cP4+x9/R1ZpltBRCCHt1BDnIcgty8WN7BtCRyGk3aNi6BmcTj0Nr85esDO2EzoKIaSd6ufQD2sHrIWfpZ/QUQhp96gYaqKCigLczL6J/o79hY5CCGnHDCWGGOQ0CPo6+kJHIaTdo2Koic6lnYOKq9Dfob/QUQgh7VxuWS6+uvUVUgpThI5CSLtGxVATXUy/CEsDS8gt5UJHIYS0c5WqSmy8uRGnUk4JHYWQdo2KoSZaFrIM24Ztg4jRW0cIeb7sjO3gZuaGP9P+FDoKIe0a/UVvIh2RDpxMnYSOQQjpIHrb98bVrKsoVZQKHYWQdosWam2C3+79hnv597CgxwJqGSKapVwBMm8B2dHAw2SgNA+w9gVGb6jav/EF4GESIJIARhaAkRXgPhDov6hqf3YUYOYI6BkL9xqIVnnR/kVsj9yOK5lX0M+xn9BxCGmXqBhqgoMJB1FYWUiFEKlSlg/cPwsUpAG93qnadvTfQOplQNcEMHepKnZMa03BEPBKVYGkUgCluUBJDqCsqNqnVgNfDwQUZYBtAODaD3DrBziFALqGrf7yiHYIkgbBTM8M6SXpQkchpN1inHOhMzxXQUFB/OrVq82+ToWqAiE/hmCiz0Qs7LGwBZKRNqmiGIg5DETsBuJOAGoFYGAOzL8HiHWqWnb0TABTe6CpC/iqlEDsESDzNpD4J5Byuer6vecCg98H1CqAqwGx5Pm8NqK1VGoVxCJxi12PMXaNcx7UYhckpI2jlqGndCv7FirVlehp01PoKERIl74ETn5YVey88DbgMxKwD6wqhICqW2LPSqwD+I6s+g8AKkuA5AuAhWfV4/hTwO8zgMA3gOAZgIm0ea+FtBktWQgRQurTqvs9jLFhjLEYxlgcY2yRhv1zGWORjLHbjLETjLFWWyX1cuZliJgI3aXdW+spiTZIuQL8OBGIPlj1uOtrwBuHgdkRwNAPAacXnl9Lja4R4DEY6Pzon7mRBeDUCzi3GljrB+x9F8iLfz7PTbRKYkEiJh2YhIsZF4WOQki7pDXFEGNMDGAjgJcByAC8yhiT1TnsBoAgznkAgN0AVrZWPh2RDvra94WJrklrPSURUl48sOt14JvBQOoVoLygarupLeAcAogE+NGx6wZM+gF47xrQ7XXgzm
```

Comment:
We are getting a `No such kernel named python2` error:

https://circleci.com/gh/poliastro/poliastro/560

Perhaps rerun everything with Python 3? :wink: